applications: asset_tracker: Add lwm2m_carrier lib support (#1996)
NordicPlayground/fw-nrfconnect-nrf: applications/asset_tracker/README.rst

	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
	set Switch 2 to the GND position.

.. _lwm2m_carrier_support:

LWM2M Carrier Library Support

> *****************************

Reviewer: ```suggestion
Using the LwM2M carrier library
*******************************

```

> This application supports the |NCS| :ref:`lwm2m_carrier` library.

Reviewer: ```suggestion
This application supports the |NCS| :ref:`liblwm2m_carrier_readme` library.
```

> To enable the LWM2M carrier library, add the following to your ``west build`` command:

Reviewer: ```suggestion
To enable the LwM2M carrier library, add the following parameter to your ``west build`` command:
```

> ``-DOVERLAY_CONFIG=lwm2m_carrier_overlay.conf``

Reviewer: Can this be done in SEGGER Embedded Studio as well, and if so, how? Or is the only way to do this in SES to set all the configuration options manually?

Author: I am not familiar enough with SES to know the answer.

	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
	set Switch 2 to the GND position.

.. _lwm2m_carrier_support:

Using the LwM2M carrier library
*******************************
This application supports the |NCS| :ref:`liblwm2m_carrier_readme` library.

> To enable the LwM2M carrier library, add the following parameter to your ``west build`` command:

Reviewer: ```suggestion
To enable the LwM2M carrier library, add the following parameter to your build command:
```

``-DOVERLAY_CONFIG=lwm2m_carrier_overlay.conf``

> In |SES|, select **Tools** > **Options** > **nRF Connect** `to add the above CMake parameter <http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#providing-cmake-options>`_.

Reviewer: ```suggestion
In |SES|, select :guilabel:`Tools` > :guilabel:`Options` > :guilabel:`nRF Connect` to add the above CMake parameter. 
See :ref:`cmake_options` for more information.
```

> Alternatively, manually set the configuration options to match the contents of the overlay config file.

Reviewer: ```suggestion
Alternatively, you can manually set the configuration options to match the contents of the overlay config file.
```
